user_guide.adoc

```diff
@@ -3148,7 +3149,7 @@ seq:
 ----
 
 and you want to use it in a file listing specification
-`filelist.ksy`. Here's how to do that:
+`filelist.ksy`, here's how to do that:
```

Comment:
Good catch, now it makes a lot more sense! But I also suggest to replace _given that_ with _if_ at the beginning of the sentence:

> For example, ~~given that~~ **if** you have a `date.ksy` file that describes the date structure _[`date.ksy` contents]_ and you want to use it in a file listing specification, here's how to do that:

Reply:
Done.